src/platform/ios.rs

```diff
@@ -23,43 +23,28 @@ pub trait WindowExtIOS {
     /// Returns a pointer to the `UIWindow` that is used by this window.
     ///
     /// The pointer will become invalid when the `Window` is destroyed.
-    fn get_uiwindow(&self) -> *mut c_void;
+    fn ui_window(&self) -> *mut c_void;
 
     /// Returns a pointer to the `UIViewController` that is used by this window.
     ///
     /// The pointer will become invalid when the `Window` is destroyed.
-    fn get_uiviewcontroller(&self) -> *mut c_void;
+    fn ui_view_controller(&self) -> *mut c_void;
 
     /// Returns a pointer to the `UIView` that is used by this window.
     ///
     /// The pointer will become invalid when the `Window` is destroyed.
-    fn get_uiview(&self) -> *mut c_void;
-
-    /// Sets the HiDpi factor used by this window.
-    /// 
-    /// This translates to `-[UIWindow setContentScaleFactor:hidpi_factor]`.
-    fn set_hidpi_factor(&self, hidpi_factor: f64);
```

Comment:
What happened to these?

Reply:
That's a good question! Looks like stuff got messed up on the iOS backend when I tried to rebase it - I'll just go ahead and re-apply them from scratch. That should be less work than figuring out how to fix this.